README.md

```diff
@@ -6,8 +6,10 @@
 [![GitHub](https://img.shields.io/github/license/keisheiled/flake8-implicit-str-concat.svg)](LICENSE)
 [![Code style: Black](https://img.shields.io/badge/code%20style-black-000000.svg)](https://github.com/psf/black)
 
-Forbid implicitly concatenated string literals on one line such as those
-introduced by [Black](https://github.com/psf/black/issues/26).
+This is a plugin for the Python code checking tool [Flake8](http://flake8.pycqa.org/)
```

Comment:
should probably repeat `Flake8 plugin to encourage correct string literal concatenation.` here somewhere

Reply:
Updated to use the encourage wording.